ibc_public/utils_relaxo.py

```diff
@@ -380,31 +515,67 @@ def t2_pipeline(do_coreg=True, do_normalise_before=False,
-                                        + '_' + fi.split('_')[-1])
+                final_plot_name = final_recon_map.split(sep)[-1].split('.')[0]
+                ext = fi.split('_')[-1]
+                final_plot_name = final_plot_name + '_' + ext
+                final_plot_name = join(SAVE_TO, final_plot_name)
                 shutil.move(plot_name, final_plot_name)
 
         if not keep_tmp:
             shutil.rmtree(join(SAVE_TO, 'tmp_t2'))
 
         time_elapsed = time.time() - start_time
-        print('\n[INFO,  t={:.2f}s] created {} \n\n'.format(time_elapsed, final_recon_map))
+        print('\n[INFO,  t={:.2f}s] created {} \n\n'.format(time_elapsed,
+                                                            final_recon_map))
         run_count = run_count + 1
 
-
-
-
-def t1_pipeline(do_normalise_before=False,
-                do_segment=True, do_normalise_after=False,
-                do_plot=True,  keep_tmp=True ,
-                sub_name='sub-11', sess_num='ses-17', 
+def t1_pipeline(closing_iter=12, do_normalise_before=False, do_segment=True,
+                do_normalise_after=False, do_plot=True, keep_tmp=True,
+                sub_name='sub-11', sess_num='ses-17',
                 root_path='/neurospin/ibc'):
-    """
-    Preprocess qMRI t1 images and then run estimation to generate t1-maps,
-    more details in scripts/qmri_README.md,
-    only one of do_normalise_before and do_normalise_after should be True,
-    both can be False
-    """
+    """ Preprocess qMRI t1 images and then run estimation to generate t1-maps,
+    more details in scripts/qmri_README.md, only one of do_normalise_before
+    and do_normalise_after should be True, both can be False
+
+    Parameters
+    ----------
+    closing_iter : int, optional
+        number of closing iteration for creating a suitable mask, by default 12
+    do_normalise_before : bool, optional
+        whether or not to normalise BEFORE estimation. So if True the input
+        image is transformed to MNI152 space. Make it True only if 
+        do_normalise_after is False and vice versa. Also, normalising before the
+        estimation is recommended over doing it after - it has been observed to
+        give better looking results for IBC subjects, by default False
+    do_segment : bool, optional
+        whether or not to segment the image. The segments are used to create a
+        mask used to exclude skull and neck during estimation. So if False,
+        estimation is done on every voxel in the input image, by default True
+    do_normalise_after : bool, optional
+        whether or not to normalise AFTER estimation. So if True the estimated
+        map is transformed to MNI152 space. Make it True only if 
+        do_normalise_before is False and vice versa. But normalising the 
+        estimated map has been observed to give spurious results for IBC
+        subjects, by default False
```

Comment:
You probably want to enforce that only one of `do_normalise_before` and `do_normalise_after` can be true.

Reply:
added assertion to do so